test/controllers/plays_controller_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class PlaysControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @track = create(:track)
+    @user = create(:user)
+    sign_in_as @user
+  end
+
+  test 'should get index with only plays for user' do
+    create(:play, track: @track, user: create(:user))
+    get plays_url
+    assert_response :success
+    body = ActiveSupport::JSON.decode response.body
```

Comment:
What is wrong with `JSON.parse`?

Reply:
We also use ActiveSupport::JSON.decode in `test/controllers/tracks_controller_test.rb`, so I've mirrored/used the same structure for these tests

Reply:
The `TracksControllerTest` uses both `JSON.parse` and `ActiveSupport::JSON.decode`. The other tests all use `JSON.parse`. If there is no clear reason to use `JSON.parse`, I think I would prefer that (it's shorter).